README.md

```diff
@@ -249,4 +276,4 @@ I am open to any feedback and contribution.
 
 Use [Discussions](https://github.com/pellared/taskflow/discussions) or write to me: *Robert Pajak* @ [Gophers Slack](https://invite.slack.golangbridge.org/).
 
-You can also create an issue or a pull request.
+You can also create an issue, or a pull request.
```

Comment:
Are you sure a comma is needed here? In Polish, it would be a grammar error. grammarly.com also states that the comma should be removed 😉

Reply:
IntelliJ wanted it :)

Reply:
There are more sentences IntelliJ wants me to fix, this one change was rather a placeholder change to do a signed commit - only to realize that all commits need to be signed.
I've got "English (US)" spelling set up in IntelliJ.

Reply:
My grammarly is also set to en-US as this usually the standard in IT

Reply:
heh, which spell-checker to trust and standardize ;)
Personally I'm very hesitant about sending text to grammarly - even if it's public technical documentation.
If you also have GoLand, we could define this as the standard to work with.

Reply:
(side note: I hope this is beyond this PR and should be addressed as a separate item)

Reply:
Agree

Reply:
You are correct https://getproofed.com/writing-tips/differences-american-british-punctuation/ 👍